docs/markdown/howtox.md

```diff
@@ -263,6 +263,20 @@ $ meson compile
 
 After these steps the resulting binary is fully optimized.
 
+## Generate extra code to write profile information
```

Comment:
This How-to entry doesn't actually tell users anything meaningful above and beyond the fact that the option exists. I think it should be removed altogether.